newIDE/app/src/MainFrame/EditorContainers/HomePage/LearnSection/index.js

```diff
@@ -39,6 +45,29 @@ export const TUTORIAL_CATEGORY_TEXTS = {
   },
 };
 
+export const secondsToMinutesAndSeconds = (seconds: number) => {
```

Comment:
Could this go to DateDisplay.js or something similar?

Reply:
I didn't know about this file, will do